samples/winapi_drivelist/main.c

```diff
@@ -56,11 +57,25 @@ int main(void)
         debugPrint("%s\n", drive);
         while(*drive++);
     }
-    debugPrint("\ndone");
 
-    while(1) {
-        Sleep(2000);
+    debugPrint("\nDone!");
+
+unmount_e:
```

Comment:
It's not clear from `unmount_e` that it also does `unmount_c` and then stops execution.
A more appropriate name would be `unloadAfterError` or something

Reply:
`cleanup` would indeed be a better name for that label, using that.